envs/example/swift/group_vars/all.yml

```diff
@@ -3,7 +3,7 @@ stack_env: example_swift
 swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
 
 undercloud_cidr: 
-  - 10.1.1.0/24
+  - cidr: 10.1.1.0/24
```

Comment:
@dlundquist Can you please elaborate?
@pwplusnick The routes will be added only when update_routes=yes is specified during run time. The default is false. So the check will not fail unless someone explicitly enables it during vagrant run.